### PR TITLE
Updates path_parsing README.md with a note about Dan

### PR DESCRIPTION
**Pull request by zanderso**

### CODE REVIEW

File: third_party/packages/path_parsing/README.md [hunk 1]
```diff
@@ -3,3 +3,12 @@
 Split from the Flutter path drawing library to create a pure Dart parsing
 library for SVG paths and code generation (without dependencies on Flutter
 runtime).
+
+## Commemoration
+
+This package was originally authored by
+[Dan Field](https://github.com/dnfield) and has been forked here
+from https://github.com/dnfield/dart_path_parsing. Dan was a member of the
```
Comment (stuartmorgan) [inline third_party/packages/path_parsing/README.md:11]: Just noticed this while duplicating: how about `[dnfield/dart_path_parsing](http://github.com/dnfield/dart_path_parsing)` so the link isn't as clunky in the rendered text?

Comment (zanderso) [reply third_party/packages/path_parsing/README.md]: Good idea. Done.

**Comment by zanderso** [comment]
> This will need a version and changelog bump so it publishes, since the README shows up on pub.dev, and thus requires a publish event.

I think I did this right, but let me know if any other updates are needed =)

**Review by domesticmouse — APPROVED** [review]

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Comment by stuartmorgan** [comment]
Looks like there's probably a latent repo tooling bug with pathifying things in `third_party`; I'll take a look tomorrow.

**Comment by stuartmorgan** [comment]
https://github.com/flutter/packages/pull/7966 should unblock this once landed.

**Pull request merged**